### PR TITLE
Implement the digest subcommand

### PR DESCRIPTION
**Pull request by raulcabello**
fixes https://github.com/kubewarden/kwctl/issues/71

### CODE REVIEW

**Review by ereslibre — APPROVED** [review]
LGTM, tried both with `registry://` and schema-less URL's (internally defaulting to `registry://`) and works fine.

**Review by flavio — CHANGES_REQUESTED** [review]
The code has to be fixed.

Also, it would be great to have something added to `e2e-tests/e2e.bats`

File: src/main.rs [hunk 1]
```diff
@@ -372,6 +373,16 @@ async fn main() -> Result<()> {
             }
             Ok(())
         }
+        Some("digest") => {
+            if let Some(matches) = matches.subcommand_matches("digest") {
+                let uri = matches.value_of("uri").unwrap();
+                let (sources, docker_config) = remote_server_options(matches)?;
+                let registry = Registry::new(docker_config.as_ref());
+                let digest = registry.manifest_digest(uri, sources.as_ref()).await?;
+                println!("{}@{}", uri, digest);
```
[the remark this answers is not in the document]
Comment (ereslibre) [reply src/main.rs:382]: I think it's fine to keep the tag in our output, and accept the policy without tag (just with sha).

I think it provides more information to the user. They can always strip the version if they want.

[the remark this answers is not in the document]
Comment (flavio) [reply src/main.rs:382]: OMG, I assumed `<image>:<tag>@sha...` was invalid. It turns out it is not.

Container image "naming" is such a mess.

Just ignore this comment then

**Comment by flavio** [comment]
No code change needed, but please provide a simple e2e test

**Comment by raulcabello** [comment]
> No code change needed, but please provide a simple e2e test

done in https://github.com/kubewarden/kwctl/pull/143/commits/904b8ed571d58d99367f14efc271b80beb7c7b0b

**Review by flavio — APPROVED** [review]
LGTM, thanks for the tests!

**Pull request merged**